spark/src/main/scala/geotrellis/spark/pyramid/Pyramid.scala

```diff
@@ -42,6 +81,43 @@ object Pyramid extends LazyLogging {
     implicit def methodToOptions(m: ResampleMethod): Options = Options(resampleMethod = m)
   }
 
+  def fromLayerReader[
+    K: AvroRecordCodec: Boundable: JsonFormat: ClassTag: SpatialComponent,
+    V <: CellGrid: ? => TilePrototypeMethods[V]: ? => TileMergeMethods[V]: AvroRecordCodec: ClassTag,
+    M: JsonFormat: Component[?, Bounds[K]]: Component[?, LayoutDefinition]
+  ](layerName: String, maxZoom: Int, minZoom: Int, layerReader: LayerReader[LayerId]): Pyramid[K, V, M] = {
```

Comment:
I don't like this interface.  Ideally, we can specify min and max zoom as `Option`s.  `None` values mean to pull the list of available zooms from the attribute store ...... except that `LayerReader`'s interface doesn't specify `val attributeStore`.  Happy to make that change to clean up this method.  Take the onus off the user to know the range of zooms.